### PR TITLE
Hacktag: Add Angular template to /examples

### PR DESCRIPTION
**Pull request by pkeavenycontentful**
## Purpose

We've gotten feedback from prospective clients that more support for Angular would help with adoption of the Contentful system. Adding an angular example is a path to this. 

## Testing steps

- cd into /examples/angular
- npm install
- ng serve
- verify the example serves successfully

### CODE REVIEW

File: examples/angular/angular-example/.angular/cache/15.2.6/angular-webpack/b5b0dc6bea5b3bba08918cc5f24bae87effb3b03/0.pack
Comment (andipaetzold) [inline examples/angular/angular-example/.angular/cache/15.2.6/angular-webpack/b5b0dc6bea5b3bba08918cc5f24bae87effb3b03/0.pack:1]: This should probably be ignored

File: package.json
Comment (andipaetzold) [inline package.json:1]: Revert this file

File: package-lock.json
Comment (andipaetzold) [inline package-lock.json:1]: Revert this file

File: examples/angular/src/app/AppConfigLocation/appConfigLocation.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div>
```
Comment (andipaetzold) [inline examples/angular/src/app/AppConfigLocation/appConfigLocation.component.html:1]: Why the nested `<div>`?  Can't we keep it clean and simply render `<p>Hello Config (AppId {{ sdk.ids.app }})</p>` which roughly matches what our TypeScript example does

File: examples/angular/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
+  },
+  "private": true,
+  "dependencies": {
```
Comment (andipaetzold) [inline examples/angular/package.json:12]: Can we pin all dependencies?

File: examples/angular/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "angular-example",
+  "version": "0.0.0",
+  "scripts": {
```
Comment (andipaetzold) [inline examples/angular/package.json:4]: Please add the scripts from TypeScript example: https://github.com/contentful/apps/blob/be8304afc8cc52b89cfded06af6e405f7f004515/examples/typescript/package.json#L21-L23

File: examples/angular/src/favicon.ico
Comment (andipaetzold) [inline examples/angular/src/favicon.ico:1]: not needed - this is an iframe. Please also remove the `<link>` from index.html

File: examples/angular/src/app/LocalHostWarning/localHostWarning.component.ts
Comment (andipaetzold) [inline examples/angular/src/app/LocalHostWarning/localHostWarning.component.ts:1]: The capital H in `localhost` triggers me :D

File: examples/angular/src/app/app.component.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import { HomeLocation } from './HomeLocation/homeLocation.component';
+import { LocalHostWarning } from './LocalHostWarning/localHostWarning.component';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  @ViewChild(DynamicComponentDirective, { static: true })
```
Comment (andipaetzold) [inline examples/angular/src/app/app.component.ts:21]: Do you need a directive here? Feels overly complex. Shouldn't an element reference be fine?

File: examples/angular/README.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# AngularExample
```
Comment (andipaetzold) [inline examples/angular/README.md:1]: This should be aligned with existing readmes: https://github.com/contentful/apps/blob/master/examples/typescript/README.md

**Review by andipaetzold — CHANGES_REQUESTED** [review]
- Does the final build work with app hosting?

**Comment by jsdalton** [comment]
Small note that we had an internal conversation on this PR and @mgoudy91 has agreed to shepherd this forward from here. 💪

**Comment by mgoudy91** [comment]
Closing for now and if we need to revisit we can reference it

**Pull request closed without merging**